README.md

```diff
@@ -15,6 +15,13 @@ Welcome to dbt-excel, the revolutionary dbt adapter that combines the rigor of d
 
 ### **Installation**
 
+Clone this repo to your local machine and change directory:
```

Comment:
@pgoslatara : Does pip install not work for you?

Reply:
`pip install dbt-excel` works.

But it doesn't bring the `dbt_project` directory to my local machine. Otherwise I'd need to run `dbt init`, manually edit the model files with the required materialisations, manually add the source excel file and create the `profiles.yml` file.

Reply:
The idea is that you use it in your own `dbt_project`, the one inside this repo is not intended to be used by `dbt-excel` users